.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   },
   rules: {
     'comma-dangle': ['error', 'always-multiline'],
+    'indent': ['error', 2, { SwitchCase: 0 }],
```

Comment:
I'm indifferent about the rule -- `npm run lint -- --fix` will fix it so *shrug*